docs/src/user-guide/configuring/index.md

```diff
@@ -47,9 +47,9 @@ All of these options give you fine-grained control over how ESLint treats your c
 
 [**Plugins**](plugins)
 
-* [Specifying Parser](./plugins#specifying-parser)
-* [Specifying Processor](./plugins#specifying-processor)
-* [Configuring Plugins](./plugins#configuring-plugins)
+* [Configuring Plugins](./plugins#configure-plugins)
```

Comment:
We still have this inconsistency of "Configuring Plugins" in the link but then it links to "Configure Plugins". The same for the following three links. Can we make these consistent?

Reply:
i kept these as they are to match the other heading links on this page. was going to change em all as part of https://github.com/eslint/eslint/issues/16578 

can do these ones now if that's your preference

Reply:
Ah okay. As long as we're tracking that task, we don't need to do them as part of this PR. 👍